packages/react-core/src/components/Accordion/Accordion.tsx

```diff
@@ -18,6 +18,8 @@ export interface AccordionProps extends React.HTMLProps<HTMLDListElement> {
   isBordered?: boolean;
   /** Display size variant. */
   displaySize?: 'default' | 'lg';
+  /** Alignment of the toggle icon. */
+  toggleAlignment?: 'start' | 'end';
```

Comment:
Just a comment, but I might refer to this as "position"? The button uses `iconPosition` for start/end. Not sure if there is a standard for naming this kind of prop.

```suggestion
  /** Position of the toggle icon. */
  togglePosition?: 'start' | 'end';
```

Reply:
That sounds much better to me! Will push an update